Marlin/src/pins/pins_RAMPS_RE_ARM.h

```diff
@@ -259,6 +259,24 @@
     //#define SHIFT_EN            P1_22  // J5-4 & AUX-4
   #endif
 
+  #if ENABLED(REPRAP_DISCOUNT_FULL_GRAPHIC_SMART_CONTROLLER) // needed on Re-ARM because of shared SPI
+    #define SDCARD_SORT_ALPHA         // Using SORT feature to keep one directory level in RAM
+                                      // When going up/down directory levels the SD card is
+                                      // accessed but the garbage/lines are removed when the
+                                      // LCD updates
+
+    #define SDSORT_LIMIT       256    // Maximum number of sorted items (10-256). Costs 27 bytes each.
+    #define FOLDER_SORTING     -1     // -1=above  0=none  1=below
+    #define SDSORT_GCODE       false  // Allow turning sorting on/off with LCD and M34 g-code.
+    #define SDSORT_USES_RAM    true   // Pre-allocate a static array for faster pre-sorting.
+    #define SDSORT_USES_STACK  false  // Prefer the stack for pre-sorting to give back some SRAM. (Negated by next 2 options.)
+    #define SDSORT_CACHE_NAMES true   // Keep sorted items in RAM longer for speedy performance. Most expensive option.
+    #define SDSORT_DYNAMIC_RAM false  // Use dynamic allocation (within SD menus). Least expensive option. Set SDSORT_LIMIT before use!
+    #define SDSORT_CACHE_VFATS 2      // Maximum number of 13-byte VFAT entries to use for sorting.
+                                      // Note: Only affects SCROLL_LONG_FILENAMES with SDSORT_CACHE_NAMES but not SDSORT_DYNAMIC_RAM.
+    #define MAX_VFAT_ENTRIES SDSORT_CACHE_VFATS
+  #endif
```

Comment:
It would be nice for users to still be able to override things like `SDSORT_GCODE` within their own configs instead of being overridden here. For that reason alone, I think it probably is best to add a Sanity Check insisting that users enable `SDCARD_SORT_ALPHA` and any other required sub-options. Then users have the option to set the optional values as they see fit.